chart/templates/clusterrole.yaml

```diff
@@ -104,9 +104,9 @@ rules:
     resources: ["services", "endpoints"]
     verbs: ["get", "watch", "list"]
   {{- end }}
-  {{- if .Values.experimental.multiNamespaceMode.enabled }}
+  {{- if .Values.sync.toHost.namespaces.enabled }}
   - apiGroups: [""]
-    resources: ["namespaces", "serviceaccounts"]
+    resources: ["namespaces", "serviceaccounts", "services", "pods", "secrets", "configmaps", "persistentvolumeclaims", "endpoints", "events"]
```

Comment:
Why is that needed, we already transform the Role in chart/templates/role to a clusterrole that should cover this

Reply:
yeah this is a leftover from when I started developing this, no longer needed. Thx! cleaning up